demos/demo_onejoint_print_position.py

```diff
@@ -1,15 +1,19 @@
 #!/usr/bin/env python3
 """Send zero-torque commands to Finger robot and print joint positions."""
+import os
 import numpy as np
+import rospkg
 
 from robot_interfaces import one_joint
 import blmc_robots
 
-home_offset = 2.241742
+# load the default config file
+config_file_path = os.path.join(
+    rospkg.RosPack().get_path("blmc_robots"), "config", "onejoint.yml")
```

Comment:
is there a  clean way of doing this without depending on ROS? otherwise, it's ok. but generally i think it would be good to keep the ros dependencies to a minimum.

Reply:
I was thinking about this for a while and did not find a nice way that is portable (i.e. does not require any local configuration on each machine). With `rospkg` all we need to do is to source `devel/setup.bash` after building (which is anyway needed, e.g. for `rosrun`).

`rospkg` is basically a Python API to access the package structure that we anyway already have since we are using catkin, so I think this is a reasonable dependency.

Reply:
ok, makes sense :)